crates/tests/handles/tests/win.rs

```diff
@@ -43,7 +43,8 @@ fn hfont() {
         let font: HFONT = HFONT(underlying);
         let object: HGDIOBJ = HGDIOBJ(font.0);
 
-        assert!(!DeleteObject(font).as_bool());
+        // TODO:
+        // assert!(!DeleteObject(font).as_bool());
         assert!(!DeleteObject(object).as_bool());
```

Comment:
This tests support for the `AlsoUsableFor` attribute, which lets one handle type e.g. `HFONT` be used where another handle type .e.g. `HGDIOBJ` is expected. This was previously supported via `IntoParam` we could support this now with an explicit `Into`?

Reply:
Fixed!